tests/components/homematicip_cloud/test_hap.py

```diff
@@ -121,3 +150,80 @@
     await hap.async_reset()
 
     assert len(hass.config_entries.async_forward_entry_unload.mock_calls) == 8
+
+
+@patch("homeassistant.helpers.temperature.display_temp", Mock(return_value=12))
+@patch("homeassistant.components.weather.WeatherEntity.state_attributes", new={})
```

Comment:
Why do we need to patch the weather entity?

Reply:
I reduced  COMPONENTS to one platform to solve this.
There is  a weather entity that is based on home.